mmcv/image/transforms/normalize.py

```diff
@@ -1,18 +1,23 @@
 # Copyright (c) Open-MMLab. All rights reserved.
+import cv2
 import numpy as np
 
 from .colorspace import bgr2rgb, rgb2bgr
 
 
 def imnormalize(img, mean, std, to_rgb=True):
     img = img.astype(np.float32)
+    mean = mean.reshape(1, -1).astype(np.float64)
+    stdinv = 1 / std.reshape(1, -1).astype(np.float64)
     if to_rgb:
         img = bgr2rgb(img)
-    return (img - mean) / std
+    return cv2.multiply(cv2.subtract(img, mean), stdinv)
```

Comment:
Need to convert it back to float32.

Reply:
maybe

Reply:
Actually no. The result keeps `float32` invariant.

Reply:
ref: https://github.com/innerlee/cvbenchmark/blob/master/cvbenchmark/img_array_normalize.py and https://github.com/innerlee/cvbenchmark#results

Reply:
Nice! You may create a top-level directory `benchmarks` and add the benchmarks there.

Reply:
will do in a separate pr